frontend/src/pages/instance/components/InstanceForm.vue

```diff
@@ -483,6 +483,12 @@ export default {
         },
         blueprintSelectionVisible () {
             return this.creatingNew && this.showFlowBlueprintSelection && !this.input.flowBlueprintId
+        },
+        heroTitle () {
+            return this.creatingNew ? (this.creatingApplication ? 'Create a new Application' : 'Create Instance') : 'Update Instance'
```

Comment:
this is never used?

Reply:
must have gotten lost somewhere. It was supposed to replace the sectionTopMenu hero prop. Fixed it